CMake/Runtime/DownloadCacher.cmake

```diff
@@ -0,0 +1,113 @@
+#
+#  DownloadCacher.cmake
+#  Copyright 2021 ItJustWorksTM
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# function for cleaning the cache
+function (clear_download_cache)
+  # delete all contents in the cached folder
+  file (REMOVE_RECURSE "${SMCE_DIR}/cached_downloads/")
+  message ("Cache has been cleared!")
```

Comment:
`STATUS "Cache cleared"` might just be more appropriate

Reply:
i fix.